python/sglang/srt/models/deepseek_v2.py

```diff
@@ -1334,17 +1334,18 @@ def __init__(
         self.tp_size = get_tensor_model_parallel_world_size()
         self.quant_config = quant_config
         self.n_share_experts_fusion = global_server_args_dict["n_share_experts_fusion"]
-        # Only Deepseek V3/R1 can use shared experts fusion optimization now.
+        # Only Deepseek V3/R1 tp mode can use shared experts fusion optimization now.
         if (
             global_server_args_dict.get("disable_shared_experts_fusion", False)
             or self.config.architectures[0] != "DeepseekV3ForCausalLM"
             or self.config.n_routed_experts != 256
             or self.config.routed_scaling_factor != 2.5
```

Comment:
nit: is it possible we do not hardcode "2.5" in the kernels etc

Reply:
done.